marshal/consumer.go

```diff
@@ -256,16 +280,25 @@ func (c *Consumer) GetCurrentLag() int64 {
 // like a load average in Unix systems: the numbers are kind of related to how much work
 // the system is doing, but by itself they don't tell you much.
 func (c *Consumer) GetCurrentLoad() int {
+	return c.GetNumActiveClaims()
+}
+
+// GetNumActiveClaims returns the number of claims actively owned by this Consumer.
+func (c *Consumer) GetNumActiveClaims() (ct int) {
 	c.lock.RLock()
 	defer c.lock.RUnlock()
 
-	ct := 0
 	for _, cl := range c.claims {
 		if cl.Claimed() {
 			ct++
 		}
 	}
-	return ct
+	return
+}
+
+// IsClaimLimtReached returns the number of claims actively owned by this Consumer.
+func (c *Consumer) IsClaimLimtReached() bool {
```

Comment:
Do you expect clients to want to call this method? I.e., is there any reason to really expose this?

Also, typo: `Limt` -> `Limit`.